calculator.js

```diff
@@ -0,0 +1,47 @@
+const operators = ['+', '-', '*', '/'];
+
+exports.create_calculator = function (operator) {
+  const numbers = [];
```

Comment:
Its probably advisable to use a less generic name here, perhaps arrayOfNumbers. This will help make its intent later on more evident